components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/public/public.html

```diff
@@ -246,8 +246,21 @@
                     }
                 })
                 .delegate("a", "dblclick", function(e) {
-                    if ($(this).parent().attr('rel')=='image') {
-                        OME.openPopup("{% url 'web_image_viewer' 0 %}".replace('/0/', "/"+$(this).attr('id').split("-")[1]+"/" ));
+                    var $this = $(this),
+                        obj_rel = $this.parent().attr('rel'),
+                        ob_id = $this.parent().attr('id'),
+                        share_id = $this.parent().attr('data-share'),
+                        popup_url = "{% url 'webindex' %}",
+                        iid;
+                    if ((obj_rel=='image') || (obj_rel=='image-locked')) {
```

Comment:
I don't think we have any other options. Just image, which can sometimes be 'image-locked' since we never removed that (we probably should).